test/tracker/controllers/FileBackedTaskManagerTest.java

```diff
@@ -0,0 +1,117 @@
+package tracker.controllers;
+
+import org.junit.jupiter.api.Test;
+import tracker.model.*;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import static org.junit.jupiter.api.Assertions.*;
+
+class FileBackedTaskManagerTest {
+    @Test
+    void shouldBePositiveIfEmptyFileSaveAndLoadSuccess () throws IOException {
+        try {
+            File file = File.createTempFile("temp","csv", new File( "/resources") );
+            File file1 = File.createTempFile("temp1","csv", new File( "/resources") );
+            FileBackedTaskManager fb = new FileBackedTaskManager(file.toPath());
+            assertNotNull(fb);
+            FileBackedTaskManager fb1 = FileBackedTaskManager.loadFromFile(file1);
+            assertNotNull(fb1);
+        } catch (IOException e) {
+            e.getStackTrace();
+        }
+    }
+
+    @Test
+    void shouldBePositiveIfTasksSaveIsCorrect() throws IOException {
+        try {
+            File file = File.createTempFile("temp","csv", new File( "/resources") );
+            Task task = new Task("task1", "desc1");
+            Epic epic = new Epic("epic1","epicDesk");
+            Subtask subtask = new Subtask("subtask1", "subtaskDesc",
+                    Status.NEW, epic.getId());
```

Comment:
Если же сделать требуемый каталог, то тесты падают. Посмотрите, пожалуйста: 


java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because "this.taskId" is null

	at tracker.model.Task.getId(Task.java:28)
	at tracker.controllers.FileBackedTaskManagerTest.shouldBePositiveIfTasksSaveIsCorrect(FileBackedTaskManagerTest.java:34)

Reply:
Тест работает, баги найдены.